technologies_deployments.md

```diff
@@ -0,0 +1,86 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+{% include JB/setup %}
+<div class="tech_deploy">
+  <div class="tech_deploy container">
+    <h2>Technologies</h2>
+    <div class="border row">
+      <div class="border col-md-4 col-sm-4" style="height:200px;">
+        <div class="panel-content">
+          <label class="_default-text" style="margin-top: 20px;">
+            <img src="./assets/themes/zeppelin/img/spark_logo.jpg" width="140px">
+          </label>
+          <label class="_hover-text">
+            Spark interpreter group which consists of five interpreters which are Spark, Pyspark, SparkR, Spark SQL, Dependency loader.<br/>
+            <a href="./docs/0.8.0-SNAPSHOT/quickstart/spark_with_zeppelin.html" class="panel-button">USE NOW <span class="glyphicon glyphicon-chevron-right"></span></a>
+          </label>                     
+        </div>
+      </div>
+      <div class="border col-md-4 col-sm-4" style="height:200px;">
+        <div class="panel-content">
+          <label class="_default-text" style="font-size:56px;">SQL</label>
+          <label class="_hover-text">
+            JDBC interpreter lets you create a JDBC connection to any data sources seamlessly. Postgresql, Mysql, MariaDB, Redshift, Apache Hive and so on.<br/>
+            <a href="./docs/0.8.0-SNAPSHOT/quickstart/sql_with_zeppelin.html" class="panel-button">USE NOW <span class="glyphicon glyphicon-chevron-right"></span></a>
+          </label>
+        </div>
+      </div>
+      <div class="border col-md-4 col-sm-4" style="height:200px;">
+        <div class="panel-content">
+          <label class="_default-text" style="margin-top:75px;">
+            <img src="./assets/themes/zeppelin/img/python-logo-generic.svg" width="200px">
+          </label>
+          <label class="_hover-text">
+            Python is supported with Zeppelin. Also integrated Matplotlib with Python & Pyspark interpreter, and Conda is now available in Zeppelin.<br/>
+            <a href="./docs/0.8.0-SNAPSHOT/quickstart/python_with_zeppelin.html" class="panel-button">USE NOW <span class="glyphicon glyphicon-chevron-right"></span></a>
+          </label>
+        </div>
+      </div>
+    </div>
+    <div class="col-md-12 col-sm-12 col-xs-12 text-center">
+      <p style="text-align:center; margin-top: 20px; font-size: 17px; color: gray; font-weight: 200; font-style: italic; padding-bottom: 0;">
+        See more details in Zeppelin supports 20+ different interpreters.
```

Comment:
10: (MINOR). can we move these kind of big inline styles into CSS? I don't mean moving all inline styles into CSS. just big inline styles like **LINE: 52** so that others can change style easily instead of modifying unfamiliar a md file.